### ISSUE TITLE
Add A Sign Up Component 

### ISSUE THREAD

**Issue by vikram-8290** [purnima143/Pluto-UI#9]
Hi @purnima143 
I want to add a Sign up component 
**Please assign me this task**

**Comment by Parsanta** [comment]
i would like to work on this please assign this to me under the label Gssoc'23

**Comment by purnima143** [comment]
Hi @vikram-8290, assigned it to you.
@Parsanta Can you check other components, which you can work on

**Comment by vikram-8290** [comment]
Yes I am working on it, but due my college internals it will take some time

**Comment by vikram-8290** [comment]
hii @purnima143 ,
i have some doubts regarding this ,it will be great if you help me so i can finish my task.
Thank you

**Comment by purnima143** [comment]
@vikram-8290 Whats your doubt?

**Comment by vikram-8290** [comment]
1. can i use some UI libraries of react for designing the Sign Up component like Material UI
2. I have only design  the react component , and not worry about its backend working

**Comment by purnima143** [comment]
1. Don't use any library, try to build from scratch.
2. You only have to design a component and validation. Eg: if creating input for password, check password is present or not before clicking on submit

**Comment by vikram-8290** [comment]
hiii @purnima143 ,
i am ready with my code , should i raise a PR or you first wanna have look at it ?

**Comment by AdityaSingh-02** [comment]
@vikram-8290 are you still working on this issue?

**Comment by vikram-8290** [comment]
Hii @AdityaSingh-02 actually I am done with it I have only raise a PR.

**Comment by vikram-8290** [comment]
hii @purnima143  i am facing some while raising a PR , if you don’t mind can i need some help.
Thank you.

**Comment by AdityaSingh-02** [comment]
What problems are you facing?

**Comment by AdityaSingh-02** [comment]
@vikram-8290 what problems are you facing?

**Comment by vikram-8290** [comment]
not able add my component file to it
